runtime/meta/ffi.cpp

```diff
@@ -20,6 +21,8 @@ extern "C" {
   return tag; \
 }
 
+  static std::vector<ffi_type *> structTypes;
```

Comment:
just to be on the safe side, can we make this thread_local static? There is at least a small possibility we may at some point have multiple threads executing K code at the same time.